.env.template

```diff
@@ -3,9 +3,10 @@ FLASK_APP=todo_app/app
 FLASK_ENV=development
 
 # Change the following values for local development.
-SECRET_KEY=sRUBY_KEY
+SECRET_KEY=RUBY_KEY
 TOKEN=RUBY_TOKEN
 BOARD_ID=board-id
 TO_DO_ID=to-do-id
 DOING_ID=doing-id
-DONE_ID=done-id
\ No newline at end of file
+DONE_ID=done-id
+FLASK_SKIP_DOTENV=True
```

Comment:
Do you want this set for users by default? In fact it's ignored anyway, since the `.env` file is only processed if this variable isn't set; so it would only find out you want to skip the dotenv after it's already loaded it. If we wanted it, we could set it manually, but here I think we can just remove it

Reply:
I've deleted the line, although the instructions on the PDF say we need to add this line?